docs/components/checkbox-group-field/index.md

```diff
@@ -199,7 +210,8 @@ Individual checkboxes can be set to read only by setting the `@isReadOnly` argum
 
 ## Attributes and Modifiers
 
```

Comment:
Should "Attributes and Modifier" sections have an "Optional" descriptor?

Reply:
I think that makes sense, as they are not "required" 🎉